blog-content/the-complete-guide-to-opentelemetry-in-next-js.md

```diff
@@ -0,0 +1,394 @@
+---
+title: The complete guide to OpenTelemetry in Next.js
+createdAt: 2025-02-10T12:00:00.000Z
+readingTime: 21
+authorFirstName: Vadim
+authorLastName: Korolik
+authorTitle: CTO @ Highlight
+authorTwitter: 'https://twitter.com/vkorolik'
+authorLinkedIn: 'https://www.linkedin.com/in/vkorolik/'
+authorGithub: 'https://github.com/Vadman97'
+authorWebsite: 'https://vadweb.us'
+authorPFP: 'https://lh3.googleusercontent.com/a-/AOh14Gh1k7XsVMGxHMLJZ7qesyddqn1y4EKjfbodEYiY=s96-c'
+tags: 'Engineering, Backend, Observability'
+metaTitle: The complete guide to OpenTelemetry in Next.js
+---
+
+```hint
+Highlight.io is an [open source](https://github.com/highlight/highlight) monitoring platform. If you’re interested in learning more, get started at [highlight.io](https://highlight.io).
```

Comment:
Nit, but "get started" doesn't sound like something I'd want to do if I'm just interested in "learning more" - maybe we should just say "Learn more or try it out at [highlight.io](https://highlight.io)"?